tests/h/views/api_groups_test.py

```diff
@@ -14,9 +14,69 @@
 pytestmark = pytest.mark.usefixtures('GroupsJSONPresenter')
 
 
+class TestGroupsWithScope(object):
+
+    def test_proxies_to_service(self, anonymous_request, list_groups_service):
```

Comment:
Hmm, it is a matter of judgement but IMO the next three tests could probably be combined into one, since there isn't any conditional logic involved.